src/bin/pgcopydb/schema.c

```diff
@@ -428,13 +429,18 @@ struct FilteringQueries listSourceTableSizeSQL[] = {
 		"    from pg_catalog.pg_class c"
 		"         join pg_catalog.pg_namespace n on c.relnamespace = n.oid"
 
+		/* include-only-schema */
+		"         left join pg_temp.filter_include_only_schema fn "
+		"                on n.nspname = fn.nspname "
+
 		/* include-only-table */
-		"         join pg_temp.filter_include_only_table inc "
-		"           on n.nspname = inc.nspname "
-		"          and c.relname = inc.relname "
+		"         left join pg_temp.filter_include_only_table ft "
+		"                on n.nspname = ft.nspname "
+		"               and c.relname = ft.relname "
```

Comment:
Why using a LEFT JOIN for an “include-only” filter implementation? I expected a JOIN here.

Reply:
Ah yeah, nice trick!